### PR TITLE
QAP-514 : myList Endp 3003 

### PR DESCRIPTION
**Pull request by PikachuSerg1**

### CODE REVIEW

File: pages/apiPages/leader_page.ts [hunk 1]
```diff
@@ -20,4 +20,23 @@ export class ApiLeadersPage {
         expect(apiRequest.ok()).toBeTruthy()
         console.log(`Leaders for period: ${period} is dispalyed`)
     }
+
+    async getMonthLeders(url: string, userToken: string ) {
```
Comment (IvanK-QX) [inline pages/apiPages/leader_page.ts:24]: @PikachuSerg1 we already have getLeders method on the ApiLeadersPage for testing this endpoint,  please modify it and add proper assertions

File: pages/apiPages/message_page.ts [hunk 2]
```diff
@@ -86,7 +88,7 @@ export class ApiMessage3003Page {
             data,
             headers: headers,
         })
-        expect(apiRequest.ok()).toBeTruthy()
+        // expect(apiRequest.ok()).toBeTruthy()
```
Comment (IvanK-QX) [inline pages/apiPages/message_page.ts:91]: @PikachuSerg1 please remove comment // and left the assertion

File: pages/apiPages/message_page.ts [hunk 3]
```diff
@@ -95,6 +97,7 @@ export class ApiMessage3003Page {
         const type = response.type
         expect(chatId).toEqual(chatId)
         expect(type).toEqual('private')
+        console.log(response)
```
Comment (IvanK-QX) [inline pages/apiPages/message_page.ts:100]: @PikachuSerg1 please remove console.log(response)

File: tests/api_tests/3000/api_profile_test.spec.ts [hunk 4]
```diff
@@ -39,6 +39,12 @@ test.describe('Profile API test', async () => {
         await api.leadersPage.getLeders(apiUrl.qaEnvUrl, user.userToken, 'month')
     })
 
+    test('Show Month Leader Name', async () => {
```
Comment (IvanK-QX) [inline tests/api_tests/3000/api_profile_test.spec.ts:42]: @PikachuSerg1 as I mention earlier, remove this test and modify existing ones

**Review by IvanK-QX — CHANGES_REQUESTED** [review]

File: pages/apiPages/message_page.ts (hunk 2 above)
Comment (PikachuSerg1) [reply pages/apiPages/message_page.ts:91]: Updated

File: tests/api_tests/3000/api_profile_test.spec.ts (hunk 4 above)
Comment (PikachuSerg1) [reply tests/api_tests/3000/api_profile_test.spec.ts:42]: Updated

**Review by IvanK-QX — APPROVED** [review]

**Pull request merged**